### DISCUSSION TITLE
Is it possible to change math.equation size inside cetz canvas context?

### DISCUSSION THREAD

**Discussion by arkadiuszwojcik** [cetz-package/cetz#587] [Q&A] [answered]
It is possible to change `math.equation` size before `#cetz.canvas` but is it possible to change math equation sizes inside `#cetz.canvas`? What if I would like to have few math formulas in `#cetz.canvas` each of them in different sizes?

**Comment by johannes-wolf** [comment, marked as answer]
Yes, this is possible. You have to put the `set math.equation(...)` calls insides the `content(...)` they are used in, though. (Because of scoping.)
